apps/src/lib/tools/jsdebugger/JsDebugger.jsx

```diff
@@ -631,7 +637,7 @@ const styles = {
     fontSize: 18,
     ':hover': {
       cursor: 'pointer',
-      color: 'white'
+      color: color.white
```

Comment:
Setting the hover color for the bottom buttons as white was the right call after looking at the code. Given that the Jira ticket specified a black hover color, put a brief note in the PR description explaining why you went with white in this case. That will help build understanding and shared context in case someone sees the change in the future and wants to adjust it to a different color.

Reply:
Thanks for the feedback! Just edited the comment and included rationale for why I went with white.